gn2/wqflask/views.py

```diff
@@ -1219,19 +1220,19 @@ def display_diffs_users():
                            files=files)
 
 
-@app.route("/genewiki/<symbol>")
-def display_generif_page(symbol):
+@app.route("/genewiki/<string:symbol>")
+def display_genewiki_page(symbol: str):
     """Fetch GeneRIF metadata from GN3 and display it"""
-    entries = requests.get(
+    wiki = requests.get(
```

Comment:
maybe handle errors here? You could call `wiki.raise_for_status()` before calling the `.json` method